debian/aarch64/buster/entry.sh

```diff
@@ -4,13 +4,21 @@ set -m
 
 function start_udev()
 {
-	which udevd
-	if [ $? == '0' ]; then
-		udevd --daemon &> /dev/null
+	if [ "$UDEV" == "on" ]; then
+		if [ "$INITSYSTEM" != "on" ]; then
+			which udevd
+			if [ $? == '0' ]; then
+				udevd --daemon &> /dev/null
+			else
+				/lib/systemd/systemd-udevd --daemon &> /dev/null
+			fi
+			udevadm trigger &> /dev/null
+		fi
 	else
-		/lib/systemd/systemd-udevd --daemon &> /dev/null
+		if [ "$INITSYSTEM" == "on" ]; then
```

Comment:
@nghiant2710 can we not have the `else` branch of this run `systemctl unmask systemd-udevd` ? because then people won't need to put it in their dockerfiles if they want to re-enable it?

Reply:
@shaunmulligan, udev service can cause the race condition between the host and the container. If users disable udev, I don't think we need to keep this running,

Reply:
@nghiant2710 okay, I think that makes sense :+1: